superbench/config/amd_mi100_z53.yaml

```diff
@@ -36,6 +36,39 @@ superbench:
         - train
       pin_memory: yes
   benchmarks:
+    kernel-launch:
+      <<: *default_local_mode
+    rccl-bw:
+      enable: true
+      modes:
+        - name: local
+          prefix: NCCL_DEBUG=INFO NCCL_IB_DISABLE=1
+      parameters:
+        maxbytes: 128M
+        iters: 50
+    mem-bw:
+      <<: *default_local_mode
+    gemm-flops:
+      <<: *default_local_mode
+      parameters:
+        m: 7680 
+        n: 8192 
+        k: 8192
+    ib-loopback:
+      enable: true
+      modes:
+        - name: local
+          prefix: PROC_RANK={proc_rank} IB_DEVICES=0
+          parallel: no
+        - name: local
```

Comment:
we do not support multiple modes currently because cannot diff metrics

Reply:
the metric in ib-loopback will be diffed by ib index, it works fine.

Reply:
if you only run 4 processes, why not `PROC_RANK={proc_rank} IB_DEVICES={proc_rank}` directly?

Reply:
it will affect the results,they'll use same numa cores

Reply:
Use？
```
proc_num: 4
PROC_RANK={proc_rank} IB_DEVICES=0,1,2,3
parallel: no
```